pkg/reconciler/reconciler.go

```diff
@@ -364,6 +334,45 @@ func (r *Reconciler) updatePipelineRunToInProgress(ctx context.Context, logger *
 	return nil
 }
 
+func (r *Reconciler) initGitProviderClient(ctx context.Context, logger *zap.SugaredLogger, repo *v1alpha1.Repository, pr *tektonv1.PipelineRun) (provider.Interface, *info.Event, error) {
+	pacInfo := r.run.Info.GetPacOpts()
+	detectedProvider, event, err := r.detectProvider(ctx, logger, pr)
+	if err != nil {
+		return nil, nil, fmt.Errorf("cannot detect provider: %w", err)
+	}
+	detectedProvider.SetPacInfo(&pacInfo)
+
+	if event.InstallationID > 0 {
```

Comment:
Non-blocking suggestion 
```suggestion
	// installation ID indicated Github App installation
	if event.InstallationID > 0 {
```